holoviews/plotting/plotly/element.py

```diff
@@ -268,8 +268,8 @@ def generate_plot(self, key, ranges):
             if figure is None:
                 figure = fig
             else:
-                figure['data'].extend(fig['data'])
-
+                figure.add_traces(fig.data)
+        print(type(self), self.projection)
```

Comment:
Would be good to remove this.

Reply:
Whoops, missed that one. removed in ed65c14